jobboard/models.py

```diff
@@ -109,5 +111,16 @@ def get_jobs_by_recruiter_id(cls, recruiter: Recruiter):
         """
         return set(cls.objects.filter(recruiter__user_id=recruiter.user.id))
 
+    @classmethod
+    def get_jobs_by_major(cls, *majors):
+        """
+        Allows students to search for a job based on one or more university majors
+        majors parameter holds Major instances
+        """
+        jobs_to_return = set()
+        for major in majors:
+            jobs_to_return.update(set(cls.objects.filter(major=major)))
```

Comment:
Instead of the for loop you can do this in one line like so:
`cls.objects.filter(major__in= majors)`

Reply:
Fixed it